modules/tinymce/src/plugins/help/main/ts/data/PluginUrls.ts

```diff
@@ -52,30 +52,29 @@ const urls = Arr.map<PartialPluginUrl, PluginUrl>([
   { key: 'wordcount', name: 'Word Count' },
   // TODO: Add other premium plugins when they are included in the website
   { key: 'a11ychecker', name: 'Accessibility Checker', type: PluginType.Premium },
-  { key: 'advcode', name: 'Advanced Code Editor', type: PluginType.Premium },
-  { key: 'advtable', name: 'Advanced Tables', type: PluginType.Premium },
-  { key: 'advtemplate', name: 'Advanced Templates', type: PluginType.Premium, slug: 'advanced-templates' },
+  { key: 'typography', name: 'Advanced Typography', type: PluginType.Premium, slug: 'advanced-typography' },
   { key: 'ai', name: 'AI Assistant', type: PluginType.Premium },
   { key: 'casechange', name: 'Case Change', type: PluginType.Premium },
   { key: 'checklist', name: 'Checklist', type: PluginType.Premium },
-  { key: 'editimage', name: 'Enhanced Image Editing', type: PluginType.Premium },
-  { key: 'footnotes', name: 'Footnotes', type: PluginType.Premium },
-  { key: 'typography', name: 'Advanced Typography', type: PluginType.Premium, slug: 'advanced-typography' },
+  { key: 'advcode', name: 'Enhanced Code Editor', type: PluginType.Premium },
   { key: 'mediaembed', name: 'Enhanced Media Embed', type: PluginType.Premium, slug: 'introduction-to-mediaembed' },
+  { key: 'advtable', name: 'Enhanced Tables', type: PluginType.Premium },
   { key: 'export', name: 'Export', type: PluginType.Premium },
+  { key: 'footnotes', name: 'Footnotes', type: PluginType.Premium },
   { key: 'formatpainter', name: 'Format Painter', type: PluginType.Premium },
   { key: 'inlinecss', name: 'Inline CSS', type: PluginType.Premium, slug: 'inline-css' },
+  { key: 'editimage', name: 'Image Editing', type: PluginType.Premium },
```

Comment:
```suggestion
  { key: 'editimage', name: 'Image Editing', type: PluginType.Premium },
  { key: 'inlinecss', name: 'Inline CSS', type: PluginType.Premium, slug: 'inline-css' },
```

Should `Image Editing` come before `Inline CSS`?

Reply:
I knew I'd miss one